motoko/hello-angular-motoko/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "hello-angular-motoko",
+  "version": "0.0.2",
+  "author": {
+    "name": "Shawn Dotey",
+    "email": "insight431@protonmail.com",
+    "url": "https://forum.dfinity.org/u/infinityio.tech"
+  },
+  "description": "An example of implementing the Internet Computer's motoko project along side Angular",
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "TypeScript",
+    "Angular",
+    "Canister"
+  ],
+  "scripts": {
+    "build": "",
+    "prebuild": "",
+    "prestart": "",
+    "init:dfx": "npm run restart:clean:dfx && npm run refresh:motoko && dfx stop",
+    "start": "npm run restart:dfx && npm run refresh:motoko && npm run serve:both && npm run print:ng",
+    "serve:both": "concurrently -k \"npm run watch:motoko\" \"ng serve && dfx stop\"",
+    "deploy:dfx": "npm run build:prod && dfx deploy --network ic --no-wallet",
+    "watch:motoko": "nodemon --watch src --ext mo -x \"npm run refresh:motoko\"",
+    "print:dfx:www": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id www)",
+    "print:dfx:motoko": "echo local IC server: http://127.0.0.1:8000/?canisterId=$(dfx canister id motoko)",
+    "generate:motoko": "dfx generate motoko",
+    "restart:dfx": "dfx stop && dfx start --background",
+    "restart:clean:dfx": "dfx stop && dfx start --background --clean",
+    "refresh:motoko": "dfx deploy motoko && npm run copy:dfx",
+    "copy:dfx": "rsync -avr .dfx/$(echo ${DFX_NETWORK:-'**'})/canisters/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --exclude='*.most' --exclude='*.ts' --delete src/declarations",
+    "build:prod": "export NODE_ENV=production && ng build --configuration production",
+    "build:dev": "export NODE_ENV=development && ng build --configuration development"
+  },
```

Comment:
I think this is nice to keep as an 'advanced' part of the example, but it should not be the only way to deploy/control the project.

Reply:
Ok, I think I may need to circle back on that one.